build.sh

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 
-pushd "$(dirname "$0")"
```

Comment:
Is there a reason to remove this? See #865.

Reply:
No was probably too hasty after removing `cd "${BUILDDIR}"`. I'll add them again.

Reply:
I have added a `cd` command.